ansible/action_plugins/tenks_schedule.py

```diff
@@ -0,0 +1,77 @@
+# Avoid shadowing of system copy module by copy action plugin.
```

Comment:
What about some pro-forma here, a copyright notice for StackHPC and ASL2 header?

Reply:
Sure - do we only need those for Python files?

Reply:
I don't have a strong opinion on that.  What does Kayobe do?

Reply:
I tend to add copyright for python files but not YAML.